### PR TITLE
Replaced old Newtonsoft.Json references

### LINKED ISSUE #3994: Newtonsoft.json Conflicts
## Description of bug
Referencing DotNetNuke.Web dll results in following build warning:

Warning		Found conflicts between different versions of the same dependent assembly. In Visual Studio, double-click this warning (or select it and press Enter) to fix the conflicts; otherwise, add the following binding redirects to the "runtime" node in the application configuration file: <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1"><dependentAssembly><assemblyIdentity name="Newtonsoft.Json" culture="neutral" publicKeyToken="30ad4fe6b2a6aeed" /><bindingRedirect oldVersion="0.0.0.0-10.0.0.0" newVersion="10.0.0.0" /></dependentAssembly></assemblyBinding>

## Steps to reproduce
Create a new project and reference the DotNetNuke.dll and DotNetNuke.Web.dll. 

## Current behavior
Build warning is generated as soon as you reference the DotNetNuke.Web.dll

## Expected behavior
No Warnings. 

## Error information
DotNetNuke.Web is internally referring to Newtonsoft.Json.dll 6.0.0 instead of the latest 10.0.0

### PR DESCRIPTION
**Pull request by valadas**
This bumps the remaining projects that were still refencing the older versions of the library to our currently distributed version.

Closes #3994

### CODE REVIEW

File: Dnn.AdminExperience/EditBar/Dnn.EditBar.Library/Dnn.EditBar.Library.csproj [hunk 1]
```diff
@@ -43,7 +43,7 @@
     <LangVersion>7</LangVersion>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+    <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\bin\Newtonsoft.Json.dll</HintPath>
```
Comment (bdukes) [inline Dnn.AdminExperience/EditBar/Dnn.EditBar.Library/Dnn.EditBar.Library.csproj:48]: Ideally this reference would also be updated to use NuGet

**Review by bdukes — APPROVED** [review]

File: Dnn.AdminExperience/EditBar/Dnn.EditBar.Library/Dnn.EditBar.Library.csproj (hunk 1 above)
Comment (valadas) [reply Dnn.AdminExperience/EditBar/Dnn.EditBar.Library/Dnn.EditBar.Library.csproj]: Well actually it was not even referenced, so I just took the reference out :)

File: Dnn.AdminExperience/EditBar/Dnn.EditBar.Library/Dnn.EditBar.Library.csproj [hunk 2]
```diff
@@ -86,6 +83,7 @@
     </AdditionalFiles>
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```
Comment (bdukes) [inline Dnn.AdminExperience/EditBar/Dnn.EditBar.Library/Dnn.EditBar.Library.csproj:86]: We don't have this `app.config`, right?
```suggestion
```

Comment (valadas) [reply Dnn.AdminExperience/EditBar/Dnn.EditBar.Library/Dnn.EditBar.Library.csproj]: Correct, just removed it from the project

**Review by mitchelsellers — APPROVED** [review]

**Pull request merged**